index.html

```diff
@@ -48,7 +48,7 @@
 	<meta name="twitter:site:id" content="@gskinner"/>
 	<meta name="twitter:title" content="RegExr: Learn, Build, & Test RegEx"/>
 	<meta name="twitter:description" content="RegExr is an online tool to learn, build, & test Regular Expressions (RegEx / RegExp)."/>
-	<meta name="twitter:image" content="https://regexr.com/assets/card.png"/>
+	<meta name="twitter:image" content="assets/card.png"/>
 	<meta name="twitter:url" content="https://regexr.com/" />
```

Comment:
Can this be `content="/"`?

Reply:
`content="/"` would be absolut, but I think `content="./"`, which is relative, should be possible.

Reply:
Very true! 👌